### ISSUE TITLE
Password reset issue - chrome autofill

### ISSUE THREAD

**Issue by bindhyeswari** [aws-amplify/amplify-js#3799]
While trying to reset the password (for cases where the username and password have been saved), chrome autofills the `Code` and `New Password` fields with the *username* and *password*.

It seems to be ignoring the `autocomplete="off"` on the `code` input.

Chrome : `Version 75.0.3770.142 (Official Build) (64-bit)`

Pretty confusing to the end user. Planning to add a custom labels onto that for now.

**Comment by elorzafe** [comment]
@bindhyeswari what framework, version and components are you using?

Thanks!

**Comment by bindhyeswari** [comment]
```

Amplify CLI version is `1.8.5`

**Comment by bindhyeswari** [comment]
```
    "aws-amplify": "^1.1.32",
    "aws-amplify-react-native": "^2.1.15",
    "expo": "^31.0.2",
    "react": "16.5.0",
    "react-native": "https://github.com/expo/react-native/archive/sdk-31.0.0.tar.gz",
    "react-navigation": "^2.18.2"
```

I am using `withAuthenticator` and the default setup.

**Comment by bindhyeswari** [comment]
```
    "aws-amplify": "^1.1.32",
    "aws-amplify-react": "^2.3.11",
    "react": "^16.8.6"
```

I am wrapping the App with a default `withAuthenticator` and am using a custom Cognito user pool.

**Comment by ZackKanter** [comment]
Having the same issue. In the hosted Amplify password reset flow, Chrome’s password manager pre-fills the "code" and "new password" fields. Since there aren't any field names / labels outside the input boxes, users keep emailing us because they can't figure out how to successfully reset their password. A few ideas: 1) prevent the password manager from kicking in on those fields / that screen, 2) add the field names outside the input boxes, or 3) splitting this into 2 steps: first, just enter code, second, create new password.

**Comment by sammartinez** [comment]
Issue resolved in #3976 PR. Closing
